lib/bpk-component-chip/src/BpkChipWrapper.js

```diff
@@ -121,11 +121,15 @@ const dynamicStyles = BpkDynamicStyleSheet.create({
   iconOutlineDisabled: {
     color: colorBlackTint05,
   },
+  textWithLeadingAccessoryView: {
+    marginStart: spacingSm,
+  },
 });
 
 type Props = {
   ...$Exact<CommonProps>,
   dismissible: boolean,
+  leadingAccessoryView: ?Element<*>,
   selected: boolean,
```

Comment:
Nitpick: Our convention is to have optional types below the non-optional ones.

```suggestion
  selected: boolean,
  leadingAccessoryView: ?Element<*>,
```

Reply:
Addressed!